views/pages/bucketlist/bucketlistResults.ejs

```diff
@@ -4,27 +4,19 @@
 <body id="bodyWelcome">
   <main>
     <div class="welcomeLinks">
-      <a href="/welcome">Go back</a>
+      <a href="/profile">Go back</a>
     </div>
 
     <h1>Your bucketlist locations</h1>
-    <button class="buttonLogin1"><a href="/bucketlist">Add new bucketlist</a></button>
+    <button class="buttonLogin1"><a href="/bucketlist">Change your bucketlist location</a></button>
     <div class="countryResults">
 
-      <!-- loop trough the schema object -->
 
-      <% if (countryView.length > 0) { %>
-      <% countryView.forEach(view => { %>
+      <h2> To where: <%= countryView.countryQuestionOne %> </h2>
+      <h2> Why there: <%= countryView.countryQuestionTwo %> </h2>
+      <h2> With who: <%= countryView.countryQuestionThree %></h2>
```

Comment:
It may be a small change, but we should change these variables called `countryQuestionOne` (and so forth) to something that describes what kind of information it holds. 'To where', 'Why there' and 'With who' could also be part of the variablename.  Maybe we can use an object with these as keys or change the questions to something a bit more clear.